pom.xml

```diff
@@ -309,23 +309,57 @@
                 <artifactId>dolphinscheduler-tools</artifactId>
                 <version>${project.version}</version>
             </dependency>
+            <dependency>
+                <groupId>org.junit</groupId>
+                <artifactId>junit-bom</artifactId>
+                <version>5.9.0</version>
```

Comment:
Is it better to set version in properties for unified version management?

Reply:
Yes, I will add it to properties. Same as that of `mockito`